### PR TITLE
LPD-20696 CommerceReturnObjectValidationRuleEngineImplTest - Returns

### PR DESCRIPTION
**Pull request by ghost**
ref https://github.com/brianchandotcom/liferay-portal/pull/152071

https://liferay.atlassian.net/browse/LPD-20282
https://liferay.atlassian.net/browse/LPD-20696

### CODE REVIEW

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/internal/object/validation/rule/test/CommerceReturnObjectValidationRuleEngineImplTest.java [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+			_commerceOrder, _user.getUserId());
+
+		_commerceOrder = _commerceOrderEngine.transitionCommerceOrder(
+			_commerceOrder, CommerceOrderConstants.ORDER_STATUS_PROCESSING,
+			_user.getUserId(), true);
+	}
+
+	@Test
+	public void testCommerceReturnAccountEntryIdObjectValidationRuleEngineImpl() {
+		Map<String, Object> execute =
```
Comment (dsitu) [inline modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/internal/object/validation/rule/test/CommerceReturnObjectValidationRuleEngineImplTest.java:125]: I think results would be a better name, as that is what is being returned,

Map<String, Object> results =

File: modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/internal/object/validation/rule/test/CommerceReturnObjectValidationRuleEngineImplTest.java [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+				).build(),
+				null);
+
+		Assert.assertTrue(
+			GetterUtil.getBoolean(execute.get("validationCriteriaMet")));
+	}
+
+	@Test
+	public void testCommerceReturnCommerceOrderIdObjectValidationRuleEngineImpl() {
+		Map<String, Object> execute =
```
Comment (dsitu) [inline modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/internal/object/validation/rule/test/CommerceReturnObjectValidationRuleEngineImplTest.java:168]: Same here lets rename it to results

File: modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/internal/object/validation/rule/test/CommerceReturnObjectValidationRuleEngineImplTest.java [hunk 3]
```diff
@@ -0,0 +1,423 @@
...
+								_commerceOrder.getCommerceOrderId()
+							).build()
+						).build()
+					).build(),
+					null);
+
+		Assert.assertTrue(
+			GetterUtil.getBoolean(execute.get("validationCriteriaMet")));
+
+		execute =
```
Comment (dsitu) [inline modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/internal/object/validation/rule/test/CommerceReturnObjectValidationRuleEngineImplTest.java:197]: Same here lets rename it to results

File: modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/internal/object/validation/rule/test/CommerceReturnObjectValidationRuleEngineImplTest.java [hunk 4]
```diff
@@ -0,0 +1,423 @@
...
+
+		Assert.assertTrue(
+			GetterUtil.getBoolean(execute.get("validationCriteriaMet")));
+	}
+
+	@Test
+	public void testCommerceReturnCommerceOrderStatusObjectValidationRuleEngineImpl()
+		throws Exception {
+
+		Map<String, Object> execute =
```
Comment (dsitu) [inline modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/internal/object/validation/rule/test/CommerceReturnObjectValidationRuleEngineImplTest.java:250]: Same here lets rename it to results

File: modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/internal/object/validation/rule/test/CommerceReturnObjectValidationRuleEngineImplTest.java [hunk 5]
```diff
@@ -0,0 +1,423 @@
...
+					).build(),
+					null);
+
+		Assert.assertTrue(
+			GetterUtil.getBoolean(execute.get("validationCriteriaMet")));
+	}
+
+	@Test
+	public void testCommerceReturnReturnStatusObjectValidationRuleEngineImpl() {
+		Map<String, Object> execute =
```
Comment (dsitu) [inline modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/internal/object/validation/rule/test/CommerceReturnObjectValidationRuleEngineImplTest.java:325]: Same here lets rename it to results

**Review by dsitu — CHANGES_REQUESTED** [review]
Just the renaming, everything else looks good

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Review by dsitu — APPROVED** [review]

**Comment by dsitu** [comment]
Failed unique test looks unrelated to this PR.

**Comment by dsitu** [comment]
ci:force:forward

**Comment by dsitu** [comment]
ci:forward:force

**Pull request closed without merging**